docs/en/classic/compute-admin-guide/access-control/rbac.adoc

```diff
@@ -1,252 +1,5 @@
 == Role-based access control for Docker Engine
 
-Prisma Cloud lets you control access to Docker commands based on group membership.
+Support for the *Docker Access Control* at *Defend > Access > Docker* and *Access User* role at *Manage > Authentication > Roles* reached End of Support in Newton Update 1 (v31.01.123).
```

Comment:
I would delete this `rbac.adoc` file and this line as this seems like a Release Note update and redirects back to open-policy-agent.adoc.
I am updating the PR with these changes.